engine/src/org/pentaho/di/trans/step/messages/messages_en_US.properties

```diff
@@ -519,3 +519,7 @@ BaseStep.Log.ReleasedServerSocketOnPort = Released server socket on port {0}
 BaseStep.Log.RunningOnSlaveServer = Running on slave server #{0}/{1}.
 BaseStep.Log.OpenedServerSocketConnectionTo = Opened a server socket connection to {0}
 BaseStep.Log.SelectedRemoteOutputStepToServer = Selected remote output step [{0}] to open a server socket to remote step [{1}].{2} on port {3}
+
+# Documentation Urls
+BaseStep.DocumentationUrl.MetaInject=https://help.pentaho.com/Documentation/7.0/0L0/0Y0/0K0/ETL_Metadata_Injection
```

Comment:
Let's make use of Const.getBaseDocUrl and only use a fragment here:  0L0/0Y0/0K0/ETL_Metadata_Injection

Also, let's not use "0L0/0Y0/0K0/ETL_Metadata_Injection" here anyway.  This really needs to go in the step itself, via annotation / documentationUrl (and using a fragment in that case will also work, as plugins are processed in a way that will automatically make use of the baseDocUrl.